azure-pipelines.yml

```diff
@@ -107,17 +107,17 @@ jobs:
           choco upgrade llvm
```

Comment:
Could you try removing this line to see if the merged fix from https://github.com/actions/runner-images/issues/8125 is in yet?

Reply:
In the log, it says:
```
choco upgrade llvm
Chocolatey v2.2.2
Upgrading the following packages:
llvm
By upgrading, you accept licenses for the packages.

You have llvm v16.0.6 installed. Version 17.0.2 is available based on your source(s).
```
So, if we just need  llvm v16, it should work.

Reply:
Another part of the toolchain might now want llvm 17 for some reason. One way to find out. :+1: